src/Compilers/CSharp/Portable/Errors/ErrorCode.cs

```diff
@@ -1484,7 +1484,7 @@ internal enum ErrorCode
         ERR_FeatureNotAvailableInVersion7_1 = 8302,
         ERR_LanguageVersionCannotHaveLeadingZeroes = 8303,
         ERR_CompilerAndLanguageVersion = 8304,
-        WRN_Experimental = 8305,
+        WRN_WindowsExperimental = 8305,
```

Comment:
Is there an even more obscure name we can use for this? e.g. UWPExperimental? (I'm not sure if the usage is limited to there.)

Reply:
We also use the "WindowsExperimental" name in `ObsoleteAttributeData.WindowsExperimental` and `ObsoleteAttributeKind.WindowsExperimental`, so I'd rather keep them all aligned.